test-d/unpromise.ts

```diff
@@ -0,0 +1,11 @@
+import {expectType, expectError} from 'tsd';
+import {Unpromise} from '..';
+
+type Str = Unpromise<Promise<string>>;
```

Comment:
Don't use acronyms.

Reply:
Oh god, I love you for saying that. :stuck_out_tongue: